Source/Dafny/Translator.cs

```diff
@@ -8020,16 +8019,126 @@ private class WFOptions
       return "Handle" + arity;
     }
 
-    public static string Apply(int arity) {
-      return "Apply" + arity;
-    }
-
-    public static string Requires(int arity) {
-      return "Requires" + arity;
-    }
-
-    public static string Reads(int arity) {
-      return "Reads" + arity;
+    // <summary>
```

Comment:
For a method `<summary>` to show up as hover text, you need to start the lines with a triple-slash. (At least that's the case in Visual Studio. I didn't check if Rider is fine with just the two.)